docs/changelog.md

```diff
@@ -10,6 +10,9 @@ sidebar_label: Changelog
 
 ### Major Features and Improvements
 
+- Added option to use `DISTINCT` with aggregation functions.
```

Comment:
```suggestion
- `DISTINCT` subclause can now be used with aggregation functions.
```

To do what? 
(we are trying to be a bit more user-oriented... so what does this enable me? )
Also: 

For example, query `COUNT(DISTINCT n.prop)` .... will do what?